bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/EaseeBindingConstants.java

```diff
@@ -171,7 +171,7 @@ public class EaseeBindingConstants {
     public static final String THING_CONFIG_BACK_PLATE_ID = "backPlateId";
     public static final String THING_CONFIG_MASTER_BACK_PLATE_ID = "masterBackPlateId";
 
-    public static final Instant OUTDATED_DATE = Instant.MIN;
+    public static final Instant OUTDATED_DATE = Instant.EPOCH;
```

Comment:
This can now be reverted since explicitly checked in `formatDate`.
```suggestion
    public static final Instant OUTDATED_DATE = Instant.MIN;
```

Reply:
I would like to keep the `Instant.EPOCH` this is still a date which is in the past. It seems to cause less trouble in general.